extensions/amp-access-laterpay/0.1/validator-amp-access-laterpay.protoascii

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-access-laterpay
+  html_format: AMP
+  tag_name: "SCRIPT"
+  spec_name: "amp-access-laterpay extension .js script"
+  mandatory_parent: "HEAD"
+  unique_warning: true
+  also_requires_tag: "amp-access extension .js script"
+  attrs: {
+    name: "async"
+    mandatory: true
+    value: ""
+  }
+  attrs: {
+    name: "custom-element"
+    mandatory: true
+    value: "amp-access-laterpay"
+    dispatch_key: true
+  }
+  attrs: {
+    name: "src"
+    mandatory: true
+    value_regex: "https://cdn\\.ampproject\\.org/v0/amp-access-laterpay-(latest|0\\.1).js"
+  }
+  attrs: {
+    name: "type"
+    value_casei: "text/javascript"
+  }
+  cdata: {
+    blacklisted_cdata_regex: {
+      regex: "."
+      error_message: "contents"
+    }
+  }
+  spec_url: "https://www.ampproject.org/docs/reference/extended/amp-access-laterpay.html"
```

Comment:
We don't have that one yet either :) Will be added by #5920 and should be auto-deployed to ampproject.org. I switched to github url for now, since it will be available sooner.